### ISSUE TITLE
RCS1155 String Comparison Bug

### ISSUE THREAD

**Issue by VisualStudioHero** [JosefPihrt/Roslynator#945]
**Product and Version Used**: 
Roslynator 4.1.1
Visual Studio 17.3.1

**Steps to Reproduce**:
I have installed Roslynator from the visual studio marketplace for Visual Studio 2022 and have configured the .editorconfig for RCS1155 string comparison analyzer.

![image](https://user-images.githubusercontent.com/111488623/185336124-b4a0b55c-b9ac-48bf-a9b2-da387d1023e7.png)

**Issue/Bug**:
The following code snippet should show a warning in the Error List to use StringComparison when comparing strings however this is not happening.
```
  if (example == "hi")
  {
       return false;
  }
```
![image](https://user-images.githubusercontent.com/111488623/185337133-3406a5eb-2e8f-412a-9fa8-1445ec3ef141.png)

**Comment by JosefPihrt** [comment]
Hi,

RCS1155 is meant to be used for cases that are in [this](https://github.com/JosefPihrt/Roslynator/blob/06de084bc6f1f96fd4c18607ee44a3c30d6169df/src/Tests.Old/Analyzers.Tests.Old/UseStringComparison.cs) file.

I think that convert

```csharp
a == b
```

into

```csharp
string.Equals(a, b, StringComparison.Ordinal)
```

is also possible but I don't think that is should enabled by default. It could be different analyzer or it could be configured with editorconfig option.

**Comment by VisualStudioHero** [comment]
I have also tried the code below in Visual Studio which did not work.
`var a = 1  `
`var b = 2  `
`if(a == b)`

I have tried RCS1155 in a later version of Rider however i noticed that once i updated Rider to the latest version (22.2.2) it has stopped working despite having the code below in the csproj.
`<TreatWarningsAsErrors>;RCS1155</> `
